modules/acknowledgements/jsx/admin/FormAdd.js

```diff
@@ -0,0 +1,84 @@
+class FormAdd extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  onAddSubmit(e) {
+    e.preventDefault();
+
+    const val = $("#" + this.props.idPrefix + "-add-input").val();
+
+    $.ajax(
+      {
+        type: "POST",
+        url: this.props.insertUrl,
+        dataType: "json",
+        data: {
+          centerId: this.props.centerId,
+          title: val
+        },
+        success: function(data) {
+          $.ajax(
+            {
+              method: "GET",
+              url: this.props.fetchUrl + "?id=" + encodeURIComponent(data.id),
+              dataType: "json",
+              success: function(d) {
+                window.dispatchEvent(
```

Comment:
I believe it you put opening curly brace of an function/object on the same line as context, eslint won't go crazy with indentation. e.g
```js
new CustomEvent(this.props.idPrefix + "-insert", {
  detail: d
})
```